python/requirements-test.txt

```diff
@@ -4,3 +4,4 @@ pandas
 pytest
 pytz
 pyuwsgi; sys.platform != 'win32' and python_version < '3.13'
+setuptools>=64
```

Comment:
While setuptools isn't required to build anymore, there are a few tests that create inline setup.py files and use them to test the building of third party libraries. It might be nice to port those off of setuptools, but I don't think that is critical for the initial PR